README.md

```diff
@@ -279,16 +279,16 @@ for details.
 
 Because Standard wraps RuboCop, they share the same [runtime
 requirements](https://github.com/rubocop-hq/rubocop#compatibility)—currently,
-that's MRI 2.2 and newer. While Standard can't avoid this runtime requirement,
-it does allow you to lint codebases that target Ruby versions older than 2.2 by
+that's MRI 2.3 and newer. While Standard can't avoid this runtime requirement,
+it does allow you to lint codebases that target Ruby versions older than 2.3 by
 narrowing the ruleset somewhat.
 
 Standard will default to telling RuboCop to target the currently running version
 of Ruby (by inspecting `RUBY_VERSION` at runtime. But if you want to lock it
 down, you can specify `ruby_version` in `.standard.yml`.
 
 ```
-ruby_version: 1.8.7
```

Comment:
Is there a reason you changed this line? You should still be able to lint 1.8.7 code (while executing rubocop under 2.3), right?